src/ontology/edits/oeo-model.omn

```diff
@@ -3680,6 +3743,61 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1791",
         OEO_00360020
     
     
+
+Class: OEO_00400012
+
+    Annotations: 
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1829
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1945",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Uncertainty of a model is a qualtity of a model that represents any departure from the unachievable ideal of complete determinism.",
+        <http://purl.obolibrary.org/obo/IAO_0000119> "https://www.researchgate.net/publication/264816512_Classifying_and_communicating_uncertainties_in_model-based_policy_analysis",
+        rdfs:comment "Uncertainty of a model is a situation of inadequate information, which can be of three sorts: inexactness, unreliability, and border with ignorance.",
+        rdfs:label "uncertainty of a model"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000019>,
+        OEO_00400037 some OEO_00400013,
+        OEO_00400038 some OEO_00400017
+    
+    
+Class: OEO_00400013
+
+    Annotations: 
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1829
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1945",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Nature of uncertainty is a quality of a model that represents the core reason for uncertainty within it.",
+        <http://purl.obolibrary.org/obo/IAO_0000119> "https://www.researchgate.net/publication/264816512_Classifying_and_communicating_uncertainties_in_model-based_policy_analysis",
+        rdfs:label "nature of uncertainty"@en
```

Comment:
Yes, I think that makes sense, since we also changed `uncertainty` to `uncertainty of a model`  for clarity.
I will add model to the label and definition of all relevant terms.